packages/protocol/scripts/bash/deploy_release_contracts.sh

```diff
@@ -1,59 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Have we double checked that this is no longer necessary and we have a different flow for deploying CELO grants? I personally don't remember what that looks like these days.

If yes, then can also remove the underlying TS script (`scripts/truffle/deploy_release_contracts.ts`).

If not, this should stay around.

Reply:
So `deploy_release_contracts.sh` was not referenced at all within monorepo, docs repo - the only remaining references are old workflow documents in Notion.

`deploy_release_contracts.ts` is actually part of `scripts/devchain.ts` which I was hesitant to remove due to other dependencies.

I went for removal of `deploy_release_contracts.sh` due to dependency on `artifacts` command: https://github.com/celo-org/celo-monorepo/pull/11369#discussion_r2068516875

Reply:
It might be worth checking with finance team about this, as they are the ones who usually handle release CELO grants

Reply:
Chatted with Marcin about this
TL;DR this script is still maintained as it's the only way to deploy RG contracts in production.

These lines, however, are redundant and deleted at this point:

```
if [[ ! -d "$CONTRACT_ARTIFACTS_DIR" ]]; then
  echo "Error: no contract artifacts found in $CONTRACT_ARTIFACTS_DIR. Use download-artifacts to obtain them, or build them locally." >&2
  exit 1
fi
```

Reply:
As we're getting rid of truffle, the question is of wether we should migrate this script or not. As this script is used every couple of months, I think the approach would be to leave this .sh file printing that this script has not been migrated to Foundry, and point people to instructions to checkout and build a commit where the truffle script was present.

If at somepoint that breaks, or there's a need to have these scripts back, we can migrate it to Foundry.

Reply:
Bring back script.
Removed outdated lines.
Added echoing of deprecation notice.
Left TODO to uncomment exiting in script in future PRs that deprecate Truffle.